python/pyspark/ml/regression.py

```diff
@@ -743,6 +743,18 @@ def treeWeights(self):
         """Return the weights for each tree"""
         return list(self._call_java("javaTreeWeights"))
 
```

Comment:
So we could do that, the thing is it might be a `DecisionTreeRegressionModel` or a `DecisionTreeClassificationModel` so we probably just want to return a list of `DecisionTreeModel` instead.